app/models/katello/content_view_version.rb

```diff
@@ -108,6 +108,12 @@ def to_s
       name
     end
 
+    def sorted_content_view_environments
+      content_view_environments = organization.readable_promotion_paths.flatten
+      content_view_environments.insert(0, organization.library)
+      content_view_environments.intersection(environments)
```

Comment:
Aren't these KTEnvironments and not Katello::ContentViewEnvironments?

Reply:
Renaming the variable will help here..The `content_view_environments` should be something like `organization_readable_environments`